schnorr_fun/src/frost.rs

```diff
@@ -1092,6 +1104,15 @@ impl crate::fun::bincode::Decode for EncodedFrostKey {
     }
 }
 
+#[cfg(feature = "bincode")]
+impl<'de> crate::fun::bincode::BorrowDecode<'de> for EncodedFrostKey {
```

Comment:
Did you try this: https://docs.rs/bincode/2.0.0-rc.3/bincode/macro.impl_borrow_decode.html?